modules/listener/variables.tf

```diff
@@ -0,0 +1,28 @@
+variable "prefix" {
+  type        = string
+  default     = "axiom-cloudwatch"
+  description = "prefix for resources, defaults to axiom-cloudwatch"
+}
+
+variable "forwarder_lambda_arn" {
+  type        = string
+  description = "The ARN of the Lambda function that forwards logs to Axiom"
+}
+
+variable "log_groups_prefix" {
+  type        = string
+  description = "The prefix of the CloudWatch log groups that will trigger the Axiom CloudWatch Forwarder Lambda."
+  default     = ""
+}
+
+variable "enable_cloudtrail" {
```

Comment:
Im a bit confused by this one, is the goal to enable CloudTrail events to be forwarded to Axiom or something else?

Reply:
The goal is to trigger the *Listener* when new log groups are created, for some reason it doesn't get triggered without Cloudtrail being enabled.

Reply:
That seems strange as I mention in https://github.com/axiomhq/axiom-cloudwatch-forwarder/pull/83#discussion_r1790000187.

How did we test this? The playground account should have gotten a trail anyway because we create those at the Org level.

Reply:
tested this in our dev account. I saw there was a trail already, but it didn't work. hmmm, now that I think of it, perhaps it was missing permissions, I will test again.